content/source/docs/cloud/api/changelog.html.md

```diff
@@ -13,10 +13,21 @@ page_id: "api-changelog"
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+
 ### 2021-08-11
 
 * ![breaking][] Security fix to [Configuration versions](./configuration-versions.html): upload-url attribute for [uploading configuration files](./configuration-versions.html#upload-configuration-files) is now only available on the create response.
 
+### 2021-07-30
+
+* Introduced Workspace Tagging
+    * Updated [Workspaces](./workspaces.html):
+        * added `tag_names` attribute.
+        * added `POST /workspaces/:workspace_id/relationships/tags`
+        * added `DELETE /workspaces/workspace-2/relationships/tags`
+    * New [Organization Tags](./organization-tags.html).
+    * Added `tags` attribute to [`tfrun`](./sentinel/import/tfrun.html.md)
```

Comment:
Oh, interesting. Not considered this as part of the changelog.

Reply:
@cam-stitt thoughts on mentioning Sentinel import changes here?

Reply:
it's just been merged, fyi https://github.com/hashicorp/terraform-website/pull/1815

Reply:
I'm okay with this, only because it's a change that is explicitly for the `terraform` plugin, which is only accessible within TFC/E.